### PR TITLE
Change parameter description in BatchNorm layer to make it more clear.

### PR DESCRIPTION
**Pull request by kartikdutt18**
Hey everyone,
I have changed the parameter description for the batch norm layer a bit. From parameter description it wasn't clear to me what units mean, hence added the word 'channels' which is more commonly used. Kindly let me know what you think.
Regards.

### CODE REVIEW

**Comment by kartikdutt18** [comment]
Thanks @rcurtin, @zoq, @birm for the review.
